Makefile

```diff
@@ -6,7 +6,7 @@ export PULUMI_DISABLE_AUTOMATIC_PLUGIN_ACQUISITION := true
 PROJECT_DIR := $(patsubst %/,%,$(dir $(abspath $(lastword $(MAKEFILE_LIST)))))
 
 install_plugins::
-	pulumi plugin install converter terraform 1.0.20
+	pulumi plugin install converter terraform 1.1.0
```

Comment:
Why is this necessary, curious? Adds to the scope of the change. Perhaps separate PR?

Reply:
Local test runs do not enforce this constraint unless you run `make`, which is difficult to do for an individual test. A lot of the pain here comes from different environments doing different things. :/

Reply:
Good point on enforcement, let us fix this.  This kind of change needs to be a separate PR otherwise it is hard to see which change comes from what IMO.

Reply:
Yeah, completely agreed.